experimental/QuadFormAndIsom/src/hermitian_miranda_morrison.jl

```diff
@@ -0,0 +1,781 @@
+###############################################################################
+#
+#  Computations of the finite quotients E_0/E^i: subsection 6.8. of [BH23]
+#  ============================================================================
+#
+#  The 5 following functions are an import of identical functions written by
+#  Tommy Hofmann on Magma.
+#
+###############################################################################
+
+# Split case
+
+function _get_quotient_split(P::Hecke.NfRelOrdIdl, i::Int)
+  OE = order(P)
+  E = nf(OE)
+  Eabs, EabstoE = Hecke.absolute_simple_field(E)
```

Comment:
```suggestion
  Eabs, EabstoE = absolute_simple_field(E)
```
at other places as well